business/MetadataDatabase/Controllers/UsersController.cs

```diff
@@ -30,10 +32,19 @@ public ActionResult<IEnumerable<UserDto>> Get()
         }
 
         // GET: api/users/5
```

Comment:
There will be some some tests (external integration tests) to add, and some other to fix.
Should I start to work on it on this branch, or will you do it ?

Reply:
I just finish the branch, You can work on it, or another one based on that branch